src/targets/python/helpers.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+var util = require('util')
+
+/**
+ * Create an string of given length filled with blank spaces
```

Comment:
I think this function documentation needs to be updated, as it's missing the second argument which is what is used to build the resulting string out of